From 52a152744b026f62a412ae10d17f3756c44a687f Mon Sep 17 00:00:00 2001
From: Mathieu Trudel-Lapierre <mathieu.trudel-lapierre@canonical.com>
Date: Fri, 14 Dec 2018 13:46:14 -0500
Subject: UBUNTU: grub-mkconfig: leave a trace of what files were sourced to
 generate the config

Patch-Name: ubuntu-mkconfig-leave-breadcrumbs.patch
Signed-off-by: Mathieu Trudel-Lapierre <mathieu.trudel-lapierre@canonical.com>
---
 util/grub-mkconfig.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/util/grub-mkconfig.in b/util/grub-mkconfig.in
index 29bdad0c1..72f1e25a0 100644
--- a/util/grub-mkconfig.in
+++ b/util/grub-mkconfig.in
@@ -162,10 +162,12 @@ if [ "x${GRUB_EARLY_INITRD_LINUX_STOCK}" = "x" ]; then
 fi
 
 if test -f ${sysconfdir}/default/grub ; then
+  gettext_printf "Sourcing file \`%s'\n" "${sysconfdir}/default/grub" 1>&2
   . ${sysconfdir}/default/grub
 fi
 for x in ${sysconfdir}/default/grub.d/*.cfg ; do
   if [ -e "${x}" ]; then
+    gettext_printf "Sourcing file \`%s'\n" "${x}" 1>&2
     . "${x}"
   fi
 done
